src/java.desktop/share/classes/javax/swing/plaf/basic/BasicMenuItemUI.java

```diff
@@ -659,6 +659,22 @@ protected void paintMenuItem(Graphics g, JComponent c,
         paintBackground(g, mi, background);
         paintCheckIcon(g, lh, lr, holdc, foreground);
         paintIcon(g, lh, lr, holdc);
+        if (UIManager.getLookAndFeel().getName().equals("Windows")
+            && (Integer.parseInt(System.getProperty("os.name").
+                replaceAll("[^0-9]", "")) >= 11)
```

Comment:
```suggestion
            && (Integer.parseInt(System.getProperty("os.name")
                .replaceAll("[^0-9]", "")) >= 11)
```
May shift `. operator` on next line for continuation.